static/app/utils/useHotkeys.tsx

```diff
@@ -1,8 +1,22 @@
-import {useCallback, useEffect, useMemo, useRef} from 'react';
+import {useCallback, useEffect, useMemo} from 'react';
 
 import {getKeyCode} from './getKeyCode';
 
-const modifierKeys = [getKeyCode('shift'), getKeyCode('alt'), getKeyCode('ctrl')];
+const isKeyPressed = (key: string, evt: KeyboardEvent): boolean => {
```

Comment:
should we add a comment here or somewhere else describing the limitation you mentioned with only one non-modifier key being recognized?